modules/markup/markdown/math/inline_parser.go

```diff
@@ -84,7 +88,7 @@ func (parser *inlineParser) Parse(parent ast.Node, block text.Reader, pc parser.
 			break
 		}
 		suceedingCharacter := line[pos]
-		if !isPunctuation(suceedingCharacter) && !(suceedingCharacter == ' ') {
+		if !isPunctuation(suceedingCharacter) && !(suceedingCharacter == ' ') && !isBracket(suceedingCharacter) {
```

Comment:
Maybe it should only check `)` because `suceedingCharacter` can't be `(`?

Reply:
Just consider that some weird patterns might exist, so I added `(`.

Maybe checking for `)` is enough.

Reply:
Thank you! Let's keep it simple to avoid side affects, then add more supports one by one 😁